cmake/ci.cmake

```diff
@@ -390,6 +398,8 @@ set(GCC_CXXFLAGS
     -Wunused-result
     -Wunused-value
     -Wunused-variable
+    -Wuse-after-free
+    -Wuse-after-free=3
```

Comment:
need both?

Reply:
Okay, that this was generated makes me feel better.  :D

Reply:
See https://github.com/nlohmann/gcc_flags